src/Esfa.Vacancy.Register.Infrastructure/Repositories/FrameworkCodeRepository.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Threading.Tasks;
+using Dapper;
+using Esfa.Vacancy.Register.Domain.Repositories;
+using Esfa.Vacancy.Register.Infrastructure.Settings;
+
+namespace Esfa.Vacancy.Register.Infrastructure.Repositories
+{
+    public class FrameworkCodeRepository : IFrameworkCodeRepository
+    {
+        private readonly IProvideSettings _settings;
+
+        private const string GetFrameworkCodesQuery = @"
+            SELECT 
+                CodeName
+            FROM 
+                dbo.ApprenticeshipFramework
+            WHERE 
+                ApprenticeshipFrameworkStatusTypeId = 1";
+
+        public FrameworkCodeRepository(IProvideSettings settings)
+        {
+            _settings = settings;
+        }
+
+        public async Task<IEnumerable<string>> GetAsync()
```

Comment:
We need to apply retry strategy here and in Standard Repository as well as Chris has missed it.